packages/x-components/src/types/origin.ts

```diff
@@ -40,12 +40,15 @@ export type QueryFeature =
  */
 export type ResultFeature =
   | 'search'
-  | 'recommendations'
-  | 'next_query_results'
+  | 'topclicked_recommendations'
+  | 'brand_recommendations'
+  | 'next_query_recommendations'
+  | 'semantic_recommendations'
   | 'partial_results'
   | 'identifier_result'
-  | 'brand_recommendations'
-  | 'semantics';
+  | 'recommendations' // TODO remove in major version. Deprecated value
+  | 'next_query_results' // TODO remove in major version. Deprecated value
+  | 'semantics'; // TODO remove in major version. Deprecated value
```

Comment:
Create a new type with this 3 and mark it as deprecated:

```
// TODO remove in major version. Deprecated value
/**
 * @deprecated Use other features instead
 */
type deprecatedFeatureNames = 'recommendations' 
  | 'next_query_results' // TODO remove in major version. Deprecated value
  | 'semantics'; // TODO remove in major version. Deprecated value
```

And use it in `ResultFeature`

```suggestion
  | deprecatedFeatureNames
```

Reply:
que fino